vernac/declaremods.ml

```diff
@@ -376,8 +376,8 @@ let do_module iter_objects i obj_dir obj_mp sobjs kobjs =
     iter_objects (i+1) prefix kobjs
   end
 
-let do_module' iter_objects i ((sp,kn),sobjs) =
-  do_module iter_objects i (dir_of_sp sp) (mp_of_kn kn) sobjs []
+let load_module' iter_objects i ((sp,kn),sobjs) =
+  load_module iter_objects i (dir_of_sp sp) (mp_of_kn kn) sobjs []
```

Comment:
I don't think this name change is correct, maybe `iter_module` works if you don't like `do`

Reply:
It is called in `load_object` and is parallel to `load_modtype`. It is not "load" in the sense of loading a module in memory but it seems to me that it is correctly loading in the sense of the load method of library objects?

Or are you suggesting that you would prefer swapping names as in:
```
let iter_module' load_object ...
```
In which case, why not, as it would make clear that it is about "loading" in the libobject sense.

Reply:
Without changing the code it looks like a generic iterator, not one specific to loading.
But we can specialize it to loading https://github.com/coq/coq/pull/19094